### PR TITLE
feat: add LoadQueryClient function

### PR DESCRIPTION
**Pull request by audtlr24**
Add `LoadQueryClient` function that can get `queryClient` from trustedStore.

Since the light block that has already been saved is used (by `NewQueryClient`), there is no need to use trusted block info as a parameter.

### CODE REVIEW

File: panacea/query_client.go [hunk 1]
```diff
@@ -112,7 +113,68 @@ func NewQueryClient(ctx context.Context, config *config.Config, info TrustedBloc
...
+	return &QueryClient{
+		RpcClient:         rpcClient,
+		LightClient:       lc,
+		interfaceRegistry: makeInterfaceRegistry(),
+		sgxLevelDB:        db,
+		mutex:             &lcMutex,
+	}, nil
+}
+
+func LoadQueryClient(ctx context.Context, config *config.Config) (*QueryClient, error) {
```
Comment (gyuguen) [inline panacea/query_client.go:55]: Among these, there is duplicate code that is quite similar to NewQueryClient.
I'll see if there's a way to improve

Comment (gyuguen) [reply panacea/query_client.go:55]: @audtlr24 I made the improved code PR. #39 
If you think it has been improved, please approve it.  :)

**Review by gyuguen — APPROVED** [review]
LGTM 👍

**Review by 0xHansLee — APPROVED** [review]
lgtm
It seems that the conflict needs to be resolved :)

**Pull request merged**